tools/linktool

```diff
@@ -361,6 +388,86 @@ def show_previous(link_uuid):
     previous_uuids = get_previous_links(link_uuid)
     print "Previous link UUIDs: {0}".format(', '.join(previous_uuids))
 
+def remove_from_chains(link_uuid):
```

Comment:
I can add the code to generate the SQL but in some cases it may not be able to execute, the reason why i left it out is that there appears to be a foreign key constraint from jobs which means that a chainlink can't be removed if it's already been used for a job and I figured that in many cases people would want to keep the job around. I can try to work out a way to detect if the chainlink has been used and print out a warning or similar in that situation?